libs/npm/core/src/hooks/api-editor/interfaces/IMinisterModel.ts

```diff
@@ -3,5 +3,6 @@ import { IOrganizationModel, ISortableModel } from '.';
 export interface IMinisterModel extends ISortableModel<number> {
   position: string;
   organizationId?: number;
+  aliases?: string;
```

Comment:
Keep the foreign key and related property together `organizationId` and `organization`.

`aliases` should never be undefined.